basics/nemo_makeleadfield.m

```diff
@@ -60,18 +60,18 @@
                     cfg.conductivity = [0.33 0.022 1.79 0.33]; % SI units, all 4 layers % <- from Oostendorp
                 case 3
                     cfg.conductivity = [0.33 0.0041 0.33]; % SI units, ignore CSF
-                    %                headmodel.cond = [0.33 0.022 0.33]; % SI units, ignore CSF % <- from Oostendorp
+                    %                cfg.conductivity = [0.33 0.022 0.33]; % SI units, ignore CSF % <- from Oostendorp
             end
-            headmodel.type = cfgnemo.headmodelstrategy;
-            headmodel = ft_convert_units(headmodel,'mm');    % Convert bnd to SI units
-            headmodel = ft_prepare_headmodel(cfg,headmodel.bnd);
-            
+            cfg.type = cfgnemo.headmodelstrategy;

```

Comment:
I am not sure about this one. AFAIK, ``ft_prepare_headmodel`` does not take ``cfg.type``.
Writing it into ``headmodel`` does not make senes either, since it is overwritten below. 
I guess that line can be deleted?

Reply:
`type` is simply for bookkeeping to keep track of how the headmodel was computed, it is not a configuration option. You're right that it gets overwritten by ft_prepare_headmodel, so it seems we can simply move `headmodel.type = cfgnemo.headmodelstrategy` to the end of the 'switch' block.

Reply:
Done.

Reply:
Should be `headmodel.type` and not `cfg.type`, so it gets retained in the headmodel structure itself. (Since it's often saved and reloaded for later use.)

Reply:
It is ``headmodel.type`` ?